docs/modules/develop/pages/guide_accessibility.adoc

```diff
@@ -0,0 +1,236 @@
+
+- Built-in alert/confirm dialogs, these are not supported e.g. by all screen readers
+- Default file input elements, these are not translated according to the currently selected language on the page but instead the language of the browser or the operating system
+- Default `<select>` elements can fail accessibility evaluations under specific devices and color schemes as they may not provide enough color contrast and this cannot be controlled through the stylesheets so they are also hard to fix
+- `<details>` and `<summary>` for "toggles" or "dropdowns" because these have problems with some screen readers and these issues can be hard to fix with such special elements
+- Any HTML element that has not been widely adopted (such as the mentioned `<details>` and `<summary>`), as accessibility tools are always behind the fast pace of browser development
+
+== How to build accessible elements?
+
+Regardless of the technology being used, a good general advice is to follow the guidelines for accessible elements by searching from the internet with "accessible X element" where you replace the "X" with what you are implementing.
+
+Some examples of accessible elements you can find through internet searches:
+
+- https://kittygiraudel.com/2020/12/10/accessible-icon-links/[Accessible icon links]
+- https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/dialog[Accessible modal dialog]
+- https://a11y-guidelines.orange.com/en/web/components-examples/dropdown-menu/[Accessible dropdown]
+- https://multiselect.vue-a11y.com/[Accessible multiselect]
+- https://alphagov.github.io/accessible-autocomplete/examples/[Accessible autocomplete]
+
+== Testing
+
+Different accessibility auditors can also have varying levels of expertise, some may notice even the smallest details in your application and some may be happy as long as you fulfil the minimum requirements. Always do your best regarding accessibility which includes ensuring the website fulfils the necessary technical requirements and testing the service as a whole or the elements that you are building with the actual tools used by people with disabilities.
+
+=== Screen readers
+
+Screen readers are mostly used by people with visual disabilities. To test the service with a screen reader, try using it with your eyes closed or blindfolded. Common software used for screen reading includes:
+
+- Windows: NVDA, JAWS
+- Apple (macOS, iOS): VoiceOver
+- Linux: ORCA
+- Android: TalkBack
+
+=== Voice control
+
+Some people may need to use their devices using speech in case they have physical disabilities that make it hard for them to use the common pointing devices, such as touch screen or mouse. To test the service with voice control, try using it without a keyboard, mouse or touch screen, just by using your mouth to speak. Common software used for voice control includes:
```

Comment:
```suggestion
Some people may need to use their devices using speech in case they have physical disabilities that make it hard for them to use the common pointing devices, such as a touch screen or a mouse. To test the service with voice control, try using it without a keyboard, mouse or touch screen, just by using your mouth to speak. Common software used for voice control includes:
```